scripts/migrate.go

```diff
@@ -21,13 +22,20 @@ const directory = "migrate"
 
 func main() {
 	migrate.LoadMigrations()
+	var pgOptions *pg.Options
 
-	db := pg.Connect(&pg.Options{
-		Addr:     "localhost:5432",
-		User:     "admin",
-		Database: "admin",
-		Password: "",
-	})
+	if len(os.Getenv("POSTGRES_URI")) > 0 {
+		pgOptions = lib.ParsePostgresURI(os.Getenv("POSTGRES_URI"))
+	} else {
+		pgOptions = &pg.Options{
+			Addr:     "localhost:5432",
+			User:     "admin",
+			Database: "admin",
+			Password: "",
+		}
+	}
```

Comment:
I generally prefer initializing this variable with the default value and just having a IF statement, as opposed to an if-else.